frontend/test/metabase/scenarios/datasets/datasets.cy.spec.js

```diff
@@ -354,13 +368,13 @@ describe("scenarios > datasets", () => {
           .type("D1");
         cy.findByLabelText("Description")
           .clear()
-          .type("Some helpful dataset description");
+          .type("foo");
         cy.button("Save").click();
```

Comment:
Just out of curiosity: does this change saves CI time? If so, I'd prefer shorter input strings in the future too

Reply:
It saves around 3s from this single test alone.

Reply:
Wow! Good to know, thank you!